src/avt/Queries/Queries/avtXRayImageQuery.C

```diff
@@ -1109,64 +1247,57 @@ avtXRayImageQuery::Execute(avtDataTree_p tree)
                 //
                 std::stringstream fileName;
                 if (outputDir != ".")
-                    fileName << outputDir.c_str() << "/";
-                fileName << baseName.str();
-                if (outputTypeIsBlueprint(outputType))
-                    fileName << ".cycle_" << std::setfill('0') << std::setw(6) << cycle;
+                    fileName << outputDir.c_str() << "/" << baseName.str();
+                if (multipleOutputFiles(outputType, numBins))
+                    fileName << ".00." << file_extensions[outputType];
```

Comment:
why just `%02d` sprintf-style here? Is that a default somewhere?

Reply:
I address this here: https://github.com/visit-dav/visit/pull/18153#discussion_r983949704

Somebody at some point wanted multiple outputs for one input, so having the extra numbers addresses that. In order for familying to work, the query needs to find a filename that looks like the one it is going to write out. Maybe it should look for both varieties when calculating a new family number?

Reply:
I don't think I was trying to suggest that. I just wanted to understand the rational for 4 chars in one place (sequential numberings), 6 in another (cycling numberings) and then 2 here (bin numberings?).

FYI, for engineering folks (probably unlikely to use this query so not too relevant), they use 6 or 8 chars for cycle. They tend to run a lot more cycles than hydro codes.

Reply:
I guess my rationale was this:
bin numberings get 2 digits because someone in the past thought that was a good idea
familying gets 4 digits because someone in the past thought it was a good idea
cycle info gets 6 digits because that seems to be what blueprint does by default.

We could definitely change these going forward.